front/app/containers/Admin/projects/components/AnalysisBanner.tsx

```diff
@@ -108,7 +108,8 @@ const AnalysisBanner = ({ projectId, phaseId }: Props) => {
             textColor={colors.teal500}
             onClick={handleGoToAnalysis}
             fontWeight="bold"
-            icon={isAnalysisAllowed ? 'stars' : 'lock'}
```

Comment:
The lock is not necessary if the feature is already visibly disabled + has a tooltip on hover.